### PR TITLE
Finalized proposal and EDA with all cells re-ran

### PR DESCRIPTION
**Pull request by elabandari**
Please review proposal updates.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,27 +8,29 @@ The original datasets and documents are sourced from the Chicago Police Departme
 
 ### 2. Research Question
 
-With our dataset we would look answer the main inferential question of whether an officer's salary influences their complaint rate (numbe of complaints per year) using a subset of the data from the years 2005 to 2015.
+With our dataset we would look answer the main inferential question of whether an police staff member's salary influences their complaint rate (number of complaints per year) using a subset of the data from the years 2005 to 2015. We suspect a relationship between these two variables
```
Comment (micahkwok) [inline README.md]: Just small grammar change, should be 'a' instead of "whether 'an' police staff member"

File: README.md [hunk 2]
```diff
@@ -8,27 +8,29 @@ The original datasets and documents are sourced from the Chicago Police Departme
...
 
 ### 4. Approach Towards Explanatory Data Analysis 
 
-Given we are joining 3 different dataframes it is important to explore what amount of information may have been lost in the process. To do this we will be observing the:  
-- Number of rows dropped during joining of data
+The first step in the EDA is to ensure that we have a thorough understanding of our dataset and that our dataset is in a form where we can easily
```
Comment (eboylestanley) [inline README.md]: Think we are missing a few words at the end of the sentence.

File: README.md [hunk 3]
```diff
@@ -8,27 +8,29 @@ The original datasets and documents are sourced from the Chicago Police Departme
...
+As the data to answer our research question is stored in 3 separate files (i.e. one file for salaries, and two files related to information about the complaints lodged against police staff members), we will have to consolidate these files by joining them through the Unique ID field. The first step in our analysis plan is merging and cleaning the dataframes. At this stage, we will also calculate the complaint rate (number of complaints per year per member) for each police staff member. Once we have cleaned our data, we will use a linear regression model to determine whether a police staff member's salary is associated with their complaint rate. The response variable for our model is the complaint rate while the explanatory variable is the salary. We have also considered other variables that may impact on complaint rate which include the staff member's age, gender and race; therefore, we will consider including these variables to our linear model if time permits. Our analysis plan may evolve as our knowledge of linear regression increases through the DSCI 561 course.
```
Comment (eboylestanley) [inline README.md]: "At this stage, we will also calculate the complaint rate (number of complaints per year per member) for each police staff 
member."  
This calculation was actually done in a separate dataframe, prior to creating the visualization.   We probably should add it as variable to the merged df, will incorporate next week.

Comment (elabandari) [reply README.md]: Great idea, I will create a Milestone 2 project and add it there.

**Review by micahkwok — APPROVED** [review]
Looks good!

**Pull request merged**